c10/util/bits16.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <cstdint>
```

Comment:
Since there are going to be a lot of bits types, it probably makes more sense to name this bits.h so that you can put them all in one repo.